### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by MrIthen** [xamarinhq/app-acquaint#1]
I'm trying to build with VS 2015 Update 2, but Acquaint.Data gives some errors. First it gave "System.Runtime.dll missing"  error, so i've added from GAC, Then gave "System.Threading.Tasks.dll is missing" and i've added it from GAC. But now it gives errors for all "ConfigureAwait" calls and "Task.FromResult" calls. What must i do?

**Comment by Freysi21** [comment]
Bump

**Comment by jsauvexamarin** [comment]
Which solution file? Acquaint.Native? Acquaint.XForms? Or Both?

**Comment by jsauvexamarin** [comment]
I just checked out a fresh working copy and it built for me.

Can you share the Installed Products details of your Visual Studio installation? (Help -> About Microsoft Visual Studio -> "Copy Info" button -> paste into text editor)

Here's mine, with the important parts **bolded**:

**Microsoft Visual Studio Community 2015
Version 14.0.25123.00 Update 2
Microsoft .NET Framework
Version 4.6.01038**

**Installed Version: Community**

Visual Basic 2015   00322-20000-00000-AA976
Microsoft Visual Basic 2015

**Visual C# 2015   00322-20000-00000-AA976
Microsoft Visual C# 2015**

Visual C++ 2015   00322-20000-00000-AA976
Microsoft Visual C++ 2015

Windows Phone SDK 8.0 - ENU   00322-20000-00000-AA976
Windows Phone SDK 8.0 - ENU

Application Insights Tools for Visual Studio Package   5.2.60328.3
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2015.1 (Beta8)   14.1.11106.0
ASP.NET and Web Tools 2015.1 (Beta8)

ASP.NET Web Frameworks and Tools 2012.2   4.1.41102.0
For additional information, visit http://go.microsoft.com/fwlink/?LinkID=309563

ASP.NET Web Frameworks and Tools 2013   5.2.40314.0
For additional information, visit http://www.asp.net/

Common Azure Tools   1.7
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

JavaScript Language Service   2.0
JavaScript Language Service

JavaScript Project System   2.0
JavaScript Project System

Microsoft Azure Mobile Services Tools   1.4
Microsoft Azure Mobile Services Tools

**NuGet Package Manager   3.4.2
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.**

PreEmptive Analytics Visualizer   1.2
Microsoft Visual Studio extension to visualize aggregated summaries from the PreEmptive Analytics product.

SQL Server Data Tools   14.0.60311.1
Microsoft SQL Server Data Tools

TypeScript   1.8.29.0
TypeScript tools for Visual Studio

**Visual Studio Tools for Universal Windows Apps   14.0.25123.00
The Visual Studio Tools for Universal Windows apps allow you to build a single universal app experience that can reach every device running Windows 10: phone, tablet, PC, and more. It includes the Microsoft Windows 10 Software Development Kit.**

**Xamarin   4.0.3.214 (0dd817c)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.**

**Xamarin.Android   6.0.3.5 (a94a03b)
Visual Studio plugin to enable development for Xamarin.Android.**

**Xamarin.iOS   9.6.1.8 (3a25bf1)
Visual Studio extension to enable development for Xamarin.iOS.**

**Comment by MrIthen** [comment]
I tried to build Acquaint.Native.sln.
I compared VS Infos, and pasted only differences below:

Microsoft Visual Studio **Professional** 2015
Version 14.0.25123.00 Update 2
Microsoft .NET Framework
Version 4.6.01038

NuGet Package Manager   **3.4.0**
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

**and below part is missing:**
Visual Studio Tools for Universal Windows Apps 14.0.25123.00
The Visual Studio Tools for Universal Windows apps allow you to build a single universal app experience that can reach every device running Windows 10: phone, tablet, PC, and more. It includes the Microsoft Windows 10 Software Development Kit.

Now i am installing the missing component (Vs Tools for Uni. Win. Apps) from the link:
http://download.microsoft.com/download/6/F/C/6FC19DBF-C223-41D0-835A-BD2155BC950D/VSToolsForWindows1C.exe

After downloading and installing and rebuilding i'll post the result.
